backends/dpdk/dpdkArch.h

```diff
@@ -596,6 +624,7 @@ class RewriteToDpdkArch : public PassManager {
         auto *evaluator = new P4::EvaluatorPass(refMap, typeMap);
         auto *parsePsa = new ParsePsa();
         info = new CollectMetadataHeaderInfo(&parsePsa->toBlockInfo);
+        passes.push_back(new ConvertActionSelector(refMap, typeMap));
```

Comment:
will this work no matter where the table is called?
e.g., what if it is called in an if or switch statement?

Reply:
Good point, in this case, since action selector table is converted into three p4 table (routing, group, member), and applied in sequence. We could transform the apply call in if or switch statement to use the apply call on the third table.